src/specs/models/work-request-bodies/WorkRequestMarkerGenes.v1.yaml

```diff
@@ -9,24 +9,10 @@ properties:
   nGenes:
     type: number
     description: Amount of marker genes to get
-  config:
-    type: object
-    properties:
-      resolution:
-        type: number
-        minimum: 0
-        maximum: 2
-        description: Resolution used to calculate the clustering. Range 0 - 2. Default 0.5 
-    required:
-      - resolution
-  type:
+  cellSetKey:
```

Comment:
why adding this?

Reply:
To ensure that `cellSetKey` is available in the request body